node_monitor/load_config.py

```diff
@@ -17,6 +17,7 @@
 DB_PASSWORD     = os.environ.get('DB_PASSWORD',     '')
 DB_NAME         = os.environ.get('DB_NAME',         '')
 DB_PORT         = os.environ.get('DB_PORT',         '')
+FEEDBACK_FORM   = os.environ.get('FEEDBACK_FORM',   '')
```

Comment:
nitpick: cool if we change this to `FEEDBACK_FORM_URL` so it's more descriptive?

Reply:
👍